checks.d/mesos_master.py

```diff
@@ -0,0 +1,246 @@
+"""Mesos Master check
+
+Collects metrics from mesos master node, only the leader is sending metrics.
+"""
+# stdlib
+from hashlib import md5
+import time
+
+# project
+from checks import AgentCheck
+
+# 3rd party
+import requests
+from version import Version
```

Comment:
Do we really need this library ? 

Python is pretty good to compare lists of integers already 

```python
[3,5,2] > [0,8,4] # True
[0, 9, 9] > [1,0,0] # False
```